preamble.tex

```diff
@@ -41,6 +42,9 @@
 \usepackage[defaultsans]{droidsans}
 \renewcommand{\familydefault}{\sfdefault}
 
+%\SetWatermarkText{Draft}
```

Comment:
One of the things the non-LaTeX rules will have a very very tough time implementing... :slightly_smiling_face: 

Thanks for doing this @dbscoach!

Reply:
True, but the watermark isn't the only hint at being draft rules and also don't get passed around and re-uploaded as much.